### ISSUE TITLE
[SCREENREADER]: IE11 + JAWS is reading all of the footer headings on first focus

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#530]
## Issue
IE11 + JAWS reads all four of the footer headings when the first link is focused by pressing `TAB`. This was flagged as a SC 4.1.1 violation. Screenshot attached.

## Audit Finding
* Note 8, Defect 1 of 3

## Acceptance Criteria
* As a screen reader user, I want to hear only the first focused link read aloud when I tab to the footer links

## Environment
* Windows 10
* IE11 + JAWS
* https://staging.va.gov/find-locations/

## Possible Fixes (optional)
The issue seems to be related to using `aria-hidden="true | false">` for the large format footer and the mobile format footer. I would recommend using classes to trigger the `display: none | block` CSS instead. By removing the aria-hidden attribute and adding inline styles to hide the mobile footer, I was able to eliminate this bug in spot testing with IE11.

## WCAG or Vendor Guidance (optional)
* [Parsing: Understanding SC 4.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)

## Screenshots or Trace Logs
![Screen Shot 2019-07-10 at 7.05.24 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/35263a70-d3f7-4505-baef-9dd031c93875)
